### PR TITLE
Adding a Glossary, closes #88.

### LINKED ISSUE #88: Add a glossary to the tutorial
It can be a list of definitions for terms like CMOR, Recipe, diagnostics, CMIP, ...

### PR DESCRIPTION
**Pull request by lopeztarifa**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [ ] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [x] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes #88 

The glossary has its own section in _extras. 

I need help to define Diagnostics and to add more terms.

### CODE REVIEW

File: _extras/glossary.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## ESMValTool Glossary
+
+Here, there are some definitions used throughout this tutorial:
+
+- **Diagnostics**:
+
+- **CMIP**: The Coupled Model Intercomparison Project (CMIP) aims at better understanding past, present and future climate changes arising from natural, unforced variability or in response to changes in radiative forcing in a multi-model context. For more information, check the [WCR-Climate webpage](https://www.wcrp-climate.org/).
+
+- **CMOR**: CMOR stands for [“Climate Model Output Rewriter” library](https://pcmdi.github.io/cmor-site/index.html). It comprises a set of C-based functions, with bindings to both Python and FORTRAN 90, that can be used to produce CF-compliant netCDF files that fulfill the requirements of many of the climate community's standard model experiments.
```
Comment (sverhoeven) [inline _extras/glossary.md:13]: In the case of ESMValTool it is more about the formatting rules defined in CMOR standard that the input files must adhere to. And the description of the variables in CMOR can be used as a reference. I think that should be part of the glossary.

File: _extras/glossary.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Glossary 
+---
+
+## ESMValTool Glossary
+
+Here, there are some definitions used throughout this tutorial:
+
+- **Diagnostics**:
```
[the remark this answers is not in the document]
Comment (lopeztarifa) [reply _extras/glossary.md]: Thanks @sverhoeven!

**Comment by stefsmeets** [comment]
I added some more suggestions to #88 of things I ran into

**Comment by stefsmeets** [comment]
Continue with this PR here: #105

**Pull request closed without merging**